src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
```

Comment:
font-family is inherited except for form elements it is better to create variables